csm/core/services/appliance_info.py

```diff
@@ -36,4 +38,6 @@ async def get_appliance_info(self):
         ret_dict = {}
         self._appliance_obj.load()
         ret_dict["serial_number"] = self._appliance_obj.get().strip()
+        cluster_id = Conf.get(const.CSM_GLOBAL_INDEX, "PROVISIONER>cluster_id")
+        ret_dict["cluster_id"] = cluster_id
```

Comment:
Do we have to raise if cluster_id not obtained... ? @pawankumarsrivastava your thought?

Reply:
This will be the case where.. csm_setup steps are not performed or `"PROVISIONER>cluster_id"` is not set deu to any reason.

Reply:
> This will be the case where.. csm_setup steps are not performed or `"PROVISIONER>cluster_id"` is not set deu to any reason.

In this case get method's default value(None) will assign to cluster_id